### PR TITLE
[FLINK-11531][docs][chinese-translation]Translate the Home Page of flink docs into Chinese.

### PR DESCRIPTION
**Pull request by klion26**
## What is the purpose of the change

Translate the Home Page of flink docs into Chinese.

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / no)
  - The serializers: (yes / no / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / no / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (yes / no / don't know)
  - The S3 file system connector: (yes / no / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
@wuchong

### CODE REVIEW

File: docs/index.zh.md [hunk 1]
```diff
@@ -24,38 +24,35 @@ under the License.
 -->
 
 
+本文档适用于 Apache Flink {{ site.version_title}} 版本。本页面最近跟新于 {% build_time %}.
```
Comment (wuchong) [inline docs/index.zh.md]:
```suggestion
本文档适用于 Apache Flink {{ site.version_title}} 版本。本页面最近更新于 {% build_time %}.
```

File: docs/index.zh.md [hunk 2]
```diff
@@ -24,38 +24,35 @@ under the License.
...
 
-This documentation is for Apache Flink version {{ site.version_title }}. These pages were built at: {% build_time %}.
+Apache Flink 是一个分布式流批一体化的开源平台。Flink 的核心是一个提供数据分发、通信以及自动容错的流计算引擎。Flink 在流计算之上构建批处理，并且原生的支持迭代计算，内存管理以及程序优化。
 
-Apache Flink is an open source platform for distributed stream and batch data processing. Flink’s core is a streaming dataflow engine that provides data distribution, communication, and fault tolerance for distributed computations over data streams. Flink builds batch processing on top of the streaming engine, overlaying native iteration support, managed memory, and program optimization.
+## 初步印象
 
-## First Steps
+- **概念**: 从 Flink 的 [数据流编程模型](concepts/programming-model.html) 和 [分布式执行环境](concepts/runtime.html) 开始了解最基本的概念。这能帮助你理解本文档的其他部分，包括如何搭建环境，进行程序编写等。建议你首先阅读此部分。
+- **Tutorials**:
```
Comment (wuchong) [inline docs/index.zh.md]:
```suggestion
- **教程**:
```

File: docs/index.zh.md [hunk 3]
```diff
@@ -24,38 +24,35 @@ under the License.
...
+  * [搭建一个本地 Flink 集群](./tutorials/local_setup.html)
 
-- **Concepts**: Start with the basic concepts of Flink's [Dataflow Programming Model](concepts/programming-model.html) and [Distributed Runtime Environment](concepts/runtime.html). This will help you understand other parts of the documentation, including the setup and programming guides. We recommend you read these sections first.
+- **编程指南**: 你可以从 [基本 API 概念](dev/api_concepts.html), [DataStream API](dev/datastream_api.html) 以及 [DataSet API](dev/batch/index.html) 着手学习如何编写你的第一个 Flink 作业。
 
-- **Tutorials**: 
-  * [Implement and run a DataStream application](./tutorials/datastream_api.html)
-  * [Setup a local Flink cluster](./tutorials/local_setup.html)
+## 部署
+在线上环境运行你的 Flink 作业之前，请阅读 [生产环境注释事项检查清单](ops/production_ready.html)。
```
Comment (wuchong) [inline docs/index.zh.md]:
```suggestion
在线上环境运行你的 Flink 作业之前，请阅读 [生产环境注意事项检查清单](ops/production_ready.html)。
```

注释 -> 注意？

File: docs/index.zh.md [hunk 4]
```diff
@@ -24,38 +24,35 @@ under the License.
...
+- **培训**: [培训资料](https://training.ververica.com//) 包含讲义，练习以及示例程序。
 
-## External Resources
-
-- **Flink Forward**: Talks from past conferences are available at the [Flink Forward](http://flink-forward.org/) website and on [YouTube](https://www.youtube.com/channel/UCY8_lgiZLZErZPF47a2hXMA). [Robust Stream Processing with Apache Flink](http://2016.flink-forward.org/kb_sessions/robust-stream-processing-with-apache-flink/) is a good place to start.
-
-- **Training**: The [training materials](http://training.data-artisans.com/) from data Artisans include slides, exercises, and sample solutions.
-
-- **Blogs**: The [Apache Flink](https://flink.apache.org/blog/) and [data Artisans](https://data-artisans.com/blog/) blogs publish frequent, in-depth technical articles about Flink.
+- **博客**: [Apache Flink](https://flink.apache.org/blog/) 以及 [ververica](https://www.ververica.com/blog) 的博客会经常更新一些有关 Flink 的技术文章。
```
Comment (wuchong) [inline docs/index.zh.md]: ververica -> Ververica

Could you also update corresponding English file for this?

File: docs/index.zh.md [hunk 5]
```diff
@@ -24,38 +24,35 @@ under the License.
...
+* [Flink 1.8 的发布日志](release-notes/flink-1.8.html)。
+* [Flink 1.7 的发布日志](release-notes/flink-1.7.html)。
+* [Flink 1.6 的发布日志](release-notes/flink-1.6.html)。
+* [Flink 1.5 的发布日志](release-notes/flink-1.5.html)。
 
-## Release Notes
+## 外部资源
 
-Release notes cover important changes between Flink versions. Please carefully read these notes if you plan to upgrade your Flink setup to a later version. 
+- **Flink Forward**: 历史大会的所有演讲均可在 [Flink Forward](http://flink-forward.org/) 官网以及 [YouTube](https://www.youtube.com/channel/UCY8_lgiZLZErZPF47a2hXMA)找到。[使用 Apache Flink 进行高可靠的流处理](http://2016.flink-forward.org/kb_sessions/robust-stream-processing-with-apache-flink/) 可以作为你第一个学习的资源。
```
Comment (wuchong) [inline docs/index.zh.md]: 历史大会的所有演讲  -> 已举办的所有大会演讲 ？

**Comment by wuchong** [comment]
Btw, I think we can simplify component name in commit message to [docs-zh]. What do you think ?

**Comment by klion26** [comment]
@wuchong thanks for your quick review. I've addressed all the comments. I think simplifing the component name to [docs-zh] is ok.

**Comment by wuchong** [comment]
Thanks @klion26  for the quick update!

Looks good to me now. 

@flinkbot approve all

**Pull request closed without merging**